docs/en/Configuring-Alluxio-with-OrangeFS.md

```diff
@@ -9,11 +9,20 @@ priority: 2
 This guide describes how to configure Alluxio with [OrangeFS](http://www.orangefs.org/) as the 
 under storage system.
 
-# Initial Setup
-
-First, the Alluxio binaries must be on your machine. You can either
-[compile Alluxio](Building-Alluxio-Master-Branch.html), or
-[download the binaries locally](Running-Alluxio-Locally.html).
+# Building Alluxio-OrangeFS Distro
+
+The integration of Alluxio-OrangeFS depends on 
+[orangefs-jni](http://www.orangefs.org/fisheye/orangefs/browse/orangefs/branches/maven-repository/maven2/org/orangefs/usrint/orangefs-jni) 
+for an optimised I/O performance that is comparable to OrangeFS 
+[native performance](http://docs.orangefs.com/v_2_9/Design_Overview.htm).
```

Comment:
I can do some benchmarks once there is a testing slot on our production cluster. However, the native-based access, [OrangeFS Direct Interface](http://docs.orangefs.com/v_2_9/ofs_client_interfaces_2_9.png), is able to bypass Linux Kernal adding lots of advantages for the integrated two-level storage system besides of performance. e.g. it can avoid the redundant page cache.